### ISSUE TITLE
Friends List

### ISSUE THREAD

**Issue by JaxonWright** [Miodec/monkeytype#1038]
The ability to have a friends list and compete with them would be awesome

**Comment by angularorbit** [comment]
Yeah, like joining a group and then having everybody's caret racing together. It can be a whole classroom!

**Comment by Miodec** [comment]
I think there are better places available (like Discord) to coordinate races with people. Maaaaaybe way into the future after I don't have anything important to do, but I'm not sure.

**Comment by wesleynw** [comment]
I agree with Miodec, I think adding friends to the site would take away from the minimalism and simplicity of the whole site. People just share their speeds through screenshots anyway in discord servers

**Comment by pjalsGit** [comment]
> I think there are better places available (like Discord) to coordinate races with people. Maaaaaybe way into the future after I don't have anything important to do, but I'm not sure.



> I agree with Miodec, I think adding friends to the site would take away from the minimalism and simplicity of the whole site. People just share their speeds through screenshots anyway in discord servers

Some people don't have Discord and therefore are disconnected from the whole world. (jk)

**Comment by timleg002** [comment]
> The ability to have a friends list and compete with them would be awesome

Interesting idea, could get around doing that, make a TODO list of some features, I'll consider

**Comment by YuetAu** [comment]
I think we can have a read-only api key and an endpoint to fetch for recent results to make a Discord bot to handle the leaderboard inside a server would be a first step to do this?

**Comment by Ferotiq** [comment]
> I think we can have a read-only api key and an endpoint to fetch for recent results to make a Discord bot to handle the leaderboard inside a server would be a first step to do this?

I don’t think that has anything to do with a friends list. The first probable step would be getting tribe done, or this could be built along with tribe. I don’t see a friends list being that useful outside of multiplayer.

**Comment by YuetAu** [comment]
Multiplayer not necessary to be in real-time. If friend list or it’s API takes time to build, a read-only key can be a equivalent for making a copy of MonkeyType Discord bot in their own Discord Server to recreate the ranking. It’s also for safety reason, although ApeKey can only fetch data for now, it might be able to change account settings later.

**Comment by Miodec** [comment]
Im gonna close it because after further thought i dont think having a friends list makes sense.

Its too much work in my opinion and communication can be (and would be even if this was made) done through much more popular and universal platforms like Discord. 

Multiplayer is still planned of course.
